### PR TITLE
SIG Node CI Subproject celebrates two years of test improvements

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**
Blog post: SIG Node CI Subproject celebrates two years of test improvements

original draft: https://docs.google.com/document/d/1uG__E94XwH0puXOAB7h-9l7xM-A8sKHaLvYGXT0NMhA/edit#

/sig node
/language en

Pre-reviewed by: @BenTheElder 

/assign @dchen1107 @derekwaynecarr 
/cc @ehashman

### CODE REVIEW

File: content/en/blog/_posts/2022-02-09-sig-node-ci-subproject-celebrates/index.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+***December 2021.*** An estimated 90% of test jobs were passing at the time of
+the 1.23 release (up from 42% in August 2020).
+
+Dockershim was removed from Kubernetes. This affected nearly half of SIG Node's
+test jobs and the SIG Node CI subproject reacted quickly and retargeted all the
+tests. SIG Node was the first SIG to complete test migrations off dockershim,
+providing examples for other affected SIGs. The vast majority of new jobs passed
+at the time of introduction without further fixes required. There are still
+some wrinkles from the dockershim removal as we uncover more dependencies on
+dockershim, but we plan to stabilize all test jobs by the 1.24 release.
```
Comment (sftim) [inline content/en/blog/_posts/2022-02-09-sig-node-ci-subproject-celebrates/index.md:128]: The removal is contentious even now. Please link to `/dockershim` (or, for the contributor site, to https://k8s.io/dockershim) with some text to let people know they can read more about that.

File: content/en/blog/_posts/2022-02-09-sig-node-ci-subproject-celebrates/index.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+- Paco Xu
+- Peter Hunt
+- Ruiwen Zhao
+- Ryan Phillips
+- Sergey Kanzhelev
+- Skyler Clark
+- Swati Sehgal
+- Wenjun Wu
+
+The kubernetes/test-infra repository contains test definitions.The number of
```
Comment (sftim) [inline content/en/blog/_posts/2022-02-09-sig-node-ci-subproject-celebrates/index.md:153]:
```suggestion
The [kubernetes/test-infra](https://github.com/kubernetes/test-infra/) source code repository contains test definitions.The number of
```

File: content/en/blog/_posts/2022-02-09-sig-node-ci-subproject-celebrates/index.md [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+- [Kubernetes SIG-Node CI Testgrid Tracker](https://docs.google.com/spreadsheets/d/1IwONkeXSc2SG_EQMYGRSkfiSWNk8yWLpVhPm-LOTbGM/edit#gid=0)
+
+We are also constantly making progress on improved tests debuggability and de-flaking.
+
+If any of this interests you, we'd love for you to join us!
+There's plenty to learn in debugging test failures, and it will help you gain
+familiarity with the code that SIG Node maintains.
+
+You can always find information about the group at the
+[SIG Node page](https://github.com/kubernetes/community/tree/master/sig-node).
```
Comment (sftim) [inline content/en/blog/_posts/2022-02-09-sig-node-ci-subproject-celebrates/index.md:182]:
```suggestion
[SIG Node](https://github.com/kubernetes/community/tree/master/sig-node) page.
```

File: content/en/blog/_posts/2022-02-09-sig-node-ci-subproject-celebrates/index.md [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+We give group updates at our maintainer track sessions:
```
Comment (sftim) [inline content/en/blog/_posts/2022-02-09-sig-node-ci-subproject-celebrates/index.md:183]:
```suggestion
We give group updates at our maintainer track sessions, such as:
```

File: content/en/blog/_posts/2022-02-09-sig-node-ci-subproject-celebrates/index.md [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+[KubeCon + CloudNativeCon Europe 2021](https://kccnceu2021.sched.com/event/iE8E/kubernetes-sig-node-intro-and-deep-dive-elana-hashman-red-hat-sergey-kanzhelev-google) and
+[KubeCon + CloudNative North America 2021](https://kccncna2021.sched.com/event/lV9D/kubenetes-sig-node-intro-and-deep-dive-elana-hashman-derek-carr-red-hat-sergey-kanzhelev-dawn-chen-google?iframe=no&w=100%&sidebar=yes&bg=no).
+Join us in our mission to keep kubelet and other SIG Node components reliable and ensure smooth and uneventful releases!
```
Comment (sftim) [inline content/en/blog/_posts/2022-02-09-sig-node-ci-subproject-celebrates/index.md:186]: (nit)
```suggestion
Join us in our mission to keep the kubelet and other SIG Node components reliable and ensure smooth and uneventful releases!
```

File: content/en/blog/_posts/2022-02-09-sig-node-ci-subproject-celebrates/index.md [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+
+Just keeping the "lights on" is a bold task and we are committed to get going.
+We are working to simplify the triage and reviews experience.
+
+Specifically, we are working on better tests organization, naming,
+and improved test tracking:
+
+- https://github.com/kubernetes/enhancements/pull/3042
+- https://github.com/kubernetes/test-infra/issues/24641
+- [Kubernetes SIG-Node CI Testgrid Tracker](https://docs.google.com/spreadsheets/d/1IwONkeXSc2SG_EQMYGRSkfiSWNk8yWLpVhPm-LOTbGM/edit#gid=0)
```
Comment (sftim) [inline content/en/blog/_posts/2022-02-09-sig-node-ci-subproject-celebrates/index.md:173]: We should add a mention of who is authorized to view this sheet (or omit it, if it's only a small audience).

File: content/en/blog/_posts/2022-02-09-sig-node-ci-subproject-celebrates/index.md [hunk 7]
```diff
@@ -0,0 +1,186 @@
...
+The subproject launched weekly hour-long meetings to discuss ongoing work
+discussion and triage.
+
+***June 2020.*** Morgan Bauer, Karan Goel, and Jorge Alarcon Ochoa were
+recognized as reviewers for the SIG Node CI group for their contributions,
+helping significantly with early stages of the subproject.
+David Porter and Roy Yang also joined the SIG test failures GitHub team.
+
+***August 2020.*** All merge- and release-blocking tests were passing,
+with some flakes. However only 42% of all SIG Node test jobs were green -- there
```
Comment (sftim) [inline content/en/blog/_posts/2022-02-09-sig-node-ci-subproject-celebrates/index.md:68]: (nit)
```suggestion
with some flakes. However only 42% of all SIG Node test jobs were green --- there
```

File: content/en/blog/_posts/2022-02-09-sig-node-ci-subproject-celebrates/index.md [hunk 8]
```diff
@@ -0,0 +1,186 @@
...
+first-time SIG Node contributors, we'd like to share some of the details and
+accomplishments over the past two years,
+helping to demystify our inner workings and celebrate the hard work
+of all our dedicated contributors!
+
+## Timeline
+
+***May 2020.*** SIG Node CI group was formed on May 11, 2020 with the goal
+to improve SIG Node CI signal and overall observability with more than
+[30 volunteers](https://docs.google.com/document/d/1fb-ugvgdSVIkkuJ388_nhp2pBTy_4HEVg5848Xy7n5U/edit#bookmark=id.vsb8pqnf4gib)
```
Comment (sftim) [inline content/en/blog/_posts/2022-02-09-sig-node-ci-subproject-celebrates/index.md:47]: Beware the extra risk of defacement sharing an “anyone with the link can suggest” document.

**Review by sftim — COMMENTED** [review]
Hi @SergeyKanzhelev 

I recommend that you submit this to https://k8s.dev/blog/ (Git repo is https://github.com/kubernetes/contributor-site)

**Keep this PR open**, though, because we can (and do) mirror articles from the contributor site blog to the main Kubernetes blog.

Would a publication date of the 16th of February be OK? You could also aim for May the 11th (the actual 2 year anniversary).

**Comment by derekwaynecarr** [comment]
great job @SergeyKanzhelev @ehashman and the whole group.

/lgtm

**Comment by sftim** [comment]
Thanks @derekwaynecarr 

The changes this needs are just enough for me to note that, but also
/lgtm cancel

File: content/en/blog/_posts/2022-02-09-sig-node-ci-subproject-celebrates/index.md (hunk 8 above)
Comment (SergeyKanzhelev) [reply content/en/blog/_posts/2022-02-09-sig-node-ci-subproject-celebrates/index.md:47]: fixed permissions, thank you!

File: content/en/blog/_posts/2022-02-09-sig-node-ci-subproject-celebrates/index.md (hunk 6 above)
Comment (SergeyKanzhelev) [reply content/en/blog/_posts/2022-02-09-sig-node-ci-subproject-celebrates/index.md:173]: Fixed permissions. everyone can view it now

File: content/en/blog/_posts/2022-02-09-sig-node-ci-subproject-celebrates/index.md [hunk 9]
```diff
@@ -0,0 +1,186 @@
+---
+layout: blog
+title: 'SIG Node CI Subproject celebrates two years of test improvements'
+date: 2022-02-09
+---
+
+# SIG Node CI Subproject celebrates two years of test improvements
```
[the remark this answers is not in the document]
Comment (SergeyKanzhelev) [reply content/en/blog/_posts/2022-02-09-sig-node-ci-subproject-celebrates/index.md:7]:
```suggestion

```

File: content/en/blog/_posts/2022-02-09-sig-node-ci-subproject-celebrates/index.md (hunk 1 above)
Comment (SergeyKanzhelev) [reply content/en/blog/_posts/2022-02-09-sig-node-ci-subproject-celebrates/index.md]: done

**Comment by SergeyKanzhelev** [comment]
@sftim done:

https://github.com/kubernetes/contributor-site/pull/288/

**Comment by nate-double-u** [comment]
I've done a copy edit pass on the contributor-site: https://github.com/kubernetes/contributor-site/pull/288#pullrequestreview-883729100

**Review by nate-double-u — APPROVED** [review]
/lgtm

**Comment by SergeyKanzhelev** [comment]
@sftim are you ok with this re-post? Or just contributor blog is better? Either works for me, I'm not sure what are the best practices

**Comment by nate-double-u** [comment]
> **Keep this PR open**, though, because we can (and do) mirror articles from the contributor site blog to the main Kubernetes blog.

Do we make any note of mirrored articles being mirrored? If not, I think that this article is ready (and [has been published on the contributor site](https://www.kubernetes.dev/blog/2022/02/16/sig-node-ci-subproject-celebrates-two-years-of-test-improvements/))

File: content/en/blog/_posts/2022-02-16-sig-node-ci-subproject-celebrates/index.md [hunk 10]
```diff
@@ -0,0 +1,190 @@
+---
+layout: blog
+title: 'SIG Node CI Subproject Celebrates Two Years of Test Improvements'
+date: 2022-02-16
+---
```
Comment (reylejano) [inline content/en/blog/_posts/2022-02-16-sig-node-ci-subproject-celebrates/index.md:5]:
```suggestion
date: 2022-02-16
slug: sig-node-ci-subproject-celebrates
canonicalUrl: https://www.kubernetes.dev/blog/2022/02/16/sig-node-ci-subproject-celebrates-two-years-of-test-improvements/
---
```

Comment (reylejano) [reply content/en/blog/_posts/2022-02-16-sig-node-ci-subproject-celebrates/index.md:5]: @SergeyKanzhelev let's add a slug and canonicalURL to the contributor site

**Comment by nate-double-u** [comment]
reapplying lgtm
/lgtm

**Pull request merged**